src/libcore/iter/range.rs

```diff
@@ -305,6 +312,11 @@ impl<A: Step> Iterator for ops::RangeFrom<A> {
         self.start = plus_n.add_one();
         Some(plus_n)
     }
+
+    #[inline]
+    fn min(self) -> Option<A> {
+        Some(self.start)
```

Comment:
Currently `min` for `RangeFrom` will either panic with overflow or loop forever so this is a change of behaviour. As `RangeFrom` is fundamentally broken right now (#25708) I don't think it's a good idea to specialise methods for it just yet.

Reply:
Yeah, I was considering not adding this yet. I'll remove it, as it would be better placed in a separate change.